common/buildings/00_tradepost.txt

```diff
@@ -463,3 +463,294 @@ trade_post = {
+	#	 local_tax_modifier = 0.2
+	#	 local_movement_speed = 0.5
+	#	 trade_route_wealth = 1
+	#
+	#	ai_creation_factor = 100
+	#}
+
+	#Bank
+	wc_bank	 = {
+		 desc = wc_bank_desc
+		 potential = {
+			always = yes
+		 }
+		 
+		 gold_cost = 600
+		 build_time = 765
+		 fort_level = 0.3
+		 tax_income = 1
+		 trade_route_wealth = 2
+		 ai_creation_factor = 100
+	}
+	
+	#GuildVault
+	wc_guildVault = {
+		desc = wc_guildVault_desc
+		potential = {
+			always = yes
+		}
+		prerequisites = { wc_bank }		
+		gold_cost = 650
+		build_time = 650
+		army_reinforce_rate = 0.1
+		retinuesize = 100
+		ai_creation_factor = 100
+	}
+	
+	#GoblinGhetto
+	wc_goblinGhetto = {
+		desc = wc_goblinGhetto_desc
+		potential = {
+			NOT = { owner = { culture_group = human_group } }
+			NOT = { owner = { culture_group = high_elf_group } }
+			NOT = { owner = { culture_group = night_elf_group } }
+		}
+		prerequisites = { wc_bank }		
+		gold_cost = 450
+		build_time = 650
+		retinuesize = 100
+		
+		garrison_size = 1
+	}
+	
+	#GoblinDistrict
+	wc_goblinDistrict = {
+		desc = wc_goblinDistrict_desc
+		potential = {
+			NOT = { owner = { culture_group = human_group } }
+			NOT = { owner = { culture_group = high_elf_group } }
+			NOT = { owner = { culture_group = night_elf_group } }
+		}
+		upgrades_from = wc_goblinGhetto		
+		gold_cost = 750
+		build_time = 750
+		retinuesize = 150
+		garrison_size = 2
+	}
+	
+		
+	#RestingLounge
+	wc_restingLounge = {
+		desc = wc_restingLounge_desc
+		potential = {
+			owner = { 
+				OR = {
+					culture_group = night_elf_group
+					culture_group = high_elf_group 
+					 
+				}
+			}
+		}
+		gold_cost = 600
+		build_time = 500
+		local_tax_modifier = 0.5
+		garrison_size = 1
+		monthly_character_piety = 0.4
+		ai_creation_factor = 100
+	}
+	
+	#Great Clan Tent
+	wc_greatClanTent = {
+		desc = wc_greatClanTent_desc
+		potential = {
+			owner = { 
+				OR = {
+					culture_group = orc_group
+					culture_group = tauren_group
+				}
+		}
+		gold_cost = 300
+		build_time = 500
+		local_tax_modifier = 0.2
+		monthly_character_piety = 0.3
+		ai_creation_factor = 100
+	}
+	
+	#Kodo Breeder
+	wc_kodoBreeder = {
+		desc = wc_kodoBreeder_desc
+		potential = {
+			owner = { culture_group = tauren_group }
+		}
+		gold_cost = 400
+		build_time = 500
+		trade_route_wealth = 0.2
+		ai_creation_factor = 100
+	}
+	
+	#MerchantVenue
+	wc_merchantVenue = {
+		desc = wc_merchantVenue_desc
+		potential = {
+			owner = { culture_group = human_group }
+		}
+		prerequisites = { wc_bank }		
+		gold_cost = 350
+		build_time = 650
+		retinuesize = 100
+		garrison_size = 1
+	}
+	
+	#Jewelcrafters
+	wc_jewelcrafters = {
+		desc = wc_jewelcrafters_desc
+		potential = {
+			port = no
+			owner = {
+				OR = {
+					culture_group = human_group 
+					culture_group = goblin_group
+					culture_group = high_elf_group
+				}
+			}
+		}	
+		gold_cost = 750
+		build_time = 650
+		tax_income = 0.5
+	}
+	
+	#Quel'dorei Jewelcrafters
+	wc_queldoreijewelcrafters = {
```

Comment:
This building ain't working?
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>